demo/admin/src/products/future/generated/CombinationFieldsTestProductsGrid.tsx

```diff
@@ -124,15 +124,15 @@ export function ProductsGrid({ toolbarAction, rowAction }: Props): React.ReactEl
             filterable: false,
             sortable: false,
             renderCell: ({ row }) => {
-                const primaryEmptyMessage = "-";
-                const typePrimaryLabels: Record<string, React.ReactNode> = {
+                const primaryTextTypeEmptyMessage = "-";
+                const primaryTextTypeLabels: Record<string, React.ReactNode> = {
```

Comment:
Why do you add the field name into the variable name?

Reply:
Should we drop the secondaryText prefix? I doubt that someone will use static selects in both primary and secondary text with different labels 😂

Reply:
Good point, I'll remove the prefix 😄